### DISCUSSION TITLE
disable logging

### DISCUSSION THREAD

**Discussion by wiryonolau** [Chia-Network/chia-blockchain#16161] [Support]
Hi is it possible to disable logging such as point to /dev/null ?
also possible to point to stdout / stderr for docker ( /proc/self/fd/... ) ?

**Comment by BrandtH22** [comment]
Hey currently the options for the logs are:
'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'

NOTSET meaning it will capture all logs.

Would you like to convert this issue into a feature request?

**Reply by wiryonolau** [reply]
Looks like pointing it to /dev/null works. 

We use this for harvester , and since there  is rarely a problem most of the time in the harvester, we want to kept minimum write.

**Reply by wiryonolau** [reply]
Looks like /dev/null have error, since chia will try to lock the file
Is there a way to disable it ...
